charts/tidb-cluster/values.yaml

```diff
@@ -77,6 +77,7 @@ pd:
   # we can only specify clusterIP and loadBalancerIP now
   service:
     clusterIP: "None"
+    # portName: "<the default name will be used>"
```

Comment:
```suggestion
      portName: "http-pd"
```

Reply:
the default value in our code is `client`, `http-pd` is the value he used in the unit tests

https://github.com/pingcap/tidb-operator/blob/7360679e33613e7c5ac16bba97a1a011ddf62d1c/pkg/manager/member/pd_member_manager.go#L446-L453

Reply:
If you do this, there will be a problem of backward compatibility. When upgrading the operator, trigger the upgrade of the cluster.

Reply:
We set value in yaml file ,but we can't change value in code.

Reply:
I agree. See my previous comments and use actual default values in comments. then people will know port name used if they didn’t set these fields. This is common practice in configuration files.

Reply:
Thanks for explaining, I understand now.